### PR TITLE
doc: add sri to cdn script tag in readme

### PR DESCRIPTION
**Pull request by shuowu-okta**

### CODE REVIEW

File: scripts/generate-sri.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  const hashBase64 = hash.digest('base64');
+  const sri = `${ALG}-${hashBase64}`;
+  return sri;
+}
+
+async function main() {
+  const argv = yargs(hideBin(process.argv)).argv;
+  const version = argv['authjs-version'];
+  if (!version) {
+    throw new Error('--authjs-version is required');
```
Comment (lesterchoi-okta) [inline scripts/generate-sri.js:100]: nit: similar comment on mono PR re; CLI args vs flags = required vs optional

**Review by lesterchoi-okta — DISMISSED** [review]
LGTM, up to you w.r.t. CLI API

File: scripts/generate-sri.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+const fs = require('fs');
+const path = require('path');
+const { spawn } = require('child_process');
+const { https } = require('follow-redirects');
```
Comment (jaredperreault-okta) [inline scripts/generate-sri.js:4]: Can this be accomplished without requiring an additional dependency?

File: scripts/generate-sri.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  if (!artifact) {
+    throw new Error('No matched version found');
+  }
+
+  if (!fs.existsSync(UNPACK_DIR)) {
+    fs.mkdirSync(UNPACK_DIR, { recursive: true });
+  }
+
+  const artifactPath = path.join(ARTIFACTS, artifact);
+  await unpackArchive(artifactPath, UNPACK_DIR);
```
Comment (jaredperreault-okta) [inline scripts/generate-sri.js:78]: The fetching of the tar and unpacking could be more easily with
`npm i --prefix <dir> @okta/okta-auth-js@X.Y.Z`

File: scripts/generate-sri.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+function generateSri(content) {
+  const hash = crypto.createHash(ALG).update(content, 'utf8');
+  const hashBase64 = hash.digest('base64');
+  const sri = `${ALG}-${hashBase64}`;
+  return sri;
+}
+
+async function main() {
+  const argv = yargs(hideBin(process.argv)).argv;
```
Comment (jaredperreault-okta) [inline scripts/generate-sri.js:97]: I don't think parsing `argv` for a single flag justifies introducing a new dependency

**Review by jaredperreault-okta — CHANGES_REQUESTED** [review]

**Comment by shuowu-okta** [comment]
Compared files from CDN and registry against v7.4.1, both are generated same sri

script used
```
const fileFromCdn = fs.readFileSync(path.join(ARTIFACTS, 'okta-auth-js-7.4.1-fromCdn.min.js'), { encoding: 'utf8' });
const fileFromPkg = fs.readFileSync(path.join(ARTIFACTS, 'okta-auth-js-7.4.1-fromPkg.min.js'), { encoding: 'utf8' });
const sriFromCdn = generateSri(fileFromCdn);
const sriFromPkg = generateSri(fileFromPkg);
console.info('generated same sri?', sriFromCdn === sriFromPkg);
```

console output:
```
shuo.wu@F7HMVX6KDM okta-auth-js % node ./scripts/generate-sri.js
Debugger attached.
generated same sri? true
```

@lesterchoi-okta @jaredperreault-okta

File: README.md [hunk 5]
```diff
@@ -96,7 +96,7 @@ require('@okta/okta-auth-js/polyfill');
 The built polyfill bundle is also available on our global CDN. Include the following script in your HTML file to load before any other scripts:
 
 ```html
-<script src="https://global.oktacdn.com/okta-auth-js/7.4.1/okta-auth-js.polyfill.js" type="text/javascript"></script>
+<script src="https://global.oktacdn.com/okta-auth-js/7.5.1/okta-auth-js.polyfill.js" type="text/javascript" integrity="sha384-EBFsuVdi4TGp/DwS7b+t+wA8zmWK10omkX05ZjJWQhzWuW31t7FWEGOnHQeIr8+L" crossorigin="anonymous"></script>
```
[the remark this answers is not in the document]
Comment (shuowu-okta) [reply README.md:99]: from cdn & yarn registry

**Review by jaredperreault-okta — APPROVED** [review]

File: scripts/setup-e2e.sh [hunk 6]
```diff
@@ -11,6 +11,7 @@ if [ -n "${TEST_SUITE_ID}" ]; then
 
   # this chrome install is not used, however it will install linux deps chrome needs (via apt-get)
   setup_service google-chrome-stable 118.0.5993.70-1
+  export PUPPETEER_DOWNLOAD_BASE_URL="https://storage.googleapis.com/chrome-for-testing-public"
```
Comment (jaredperreault-okta) [inline scripts/setup-e2e.sh:14]: Is this still necessary?

**Review by lesterchoi-okta — COMMENTED** [review]
LGTM pending @jaredperreault-okta's comment re;  `PUPPETEER_DOWNLOAD_BASE_URL`

**Review by jaredperreault-okta — APPROVED** [review]

**Pull request merged**